quirks/text-decoration-doesnt-propagate-into-tables/quirks.html

```diff
@@ -0,0 +1,14 @@
+<title>text decoration doesn't propagate into tables quirk</title>
+<link rel=match href=text-decoration-doesnt-propagate-into-tables-ref.html>
```

Comment:
This needs to point to quirks-ref.html I think. That's also why the build script failed.

Reply:
Oops. Fixed.